hack/verify-versions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Can we hook this up to the [lint job on CI](https://github.com/Kong/kubernetes-ingress-controller/blob/119dca97739dafee5f0539b96774144754328e8d/.github/workflows/test.yaml#L15)?

Reply:
It's invoking verify.manifests: https://github.com/Kong/kubernetes-ingress-controller/blob/119dca97739dafee5f0539b96774144754328e8d/.github/workflows/test.yaml#L44

Reply:
Ah, I overlooked that. Sure.